tools/integration-tests/tester/tests/mana/mana_test.go

```diff
@@ -0,0 +1,47 @@
+package mana
+
+import (
+	"testing"
+	"time"
+
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/tests"
+	"github.com/stretchr/testify/require"
+)
+
+func TestManaPersistence(t *testing.T) {
+	n, err := f.CreateNetwork("mana_TestPersistence", 1, 0, true)
```

Comment:
let's create at least 5 nodes. If you take a look at the faucet tests, there is usually 1 out of 5 that fails the faucet persistence test for the same reason this test should fail

Reply:
yeah but that shouldn't matter here. This tests that `a` node persists its mana and it starts up with the stored values.
Fixing #749 is another thing.

Reply:
okay